templates/advanced/api/services/Ember.js

```diff
@@ -39,9 +39,9 @@ var Ember = {
 		sideload = sideload || false;
 		var plural = Array.isArray( records ) ? true : false;
 
-		var emberModelIdentity = _.camelCase( model.globalId );
+		var emberModelIdentity = model.globalId;
```

Comment:
Ah, forget about my last comment. It doesn't need to be camelCased, but the `via` property has to be used.

The fix would be to replace `rec [ emberModelIdentity ] ` with `rec [ assoc.via ]` in lines 87 and 92.
@aars Do you want to put it into this PR or should I take of it myself?